README.md

```diff
@@ -1,75 +1,85 @@
-# FeedHenry Android SDK [![Build Status](https://travis-ci.org/feedhenry/fh-android-sdk.png)](https://travis-ci.org/feedhenry/fh-android-sdk)
+# FeedHenry Android SDK
 
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.feedhenry/fh-android-sdk/badge.svg?style=flat-square)](https://maven-badges.herokuapp.com/maven-central/com.feedhenry/fh-android-sdk/)
+[![Travis](https://img.shields.io/travis/feedhenry/fh-android-sdk.svg)](http://travis-ci.org/feedhenry/fh-android-sdk)
+[![Coveralls](https://img.shields.io/coveralls/feedhenry/fh-android-sdk/master.svg)](https://coveralls.io/github/feedhenry/fh-android-sdk)
+[![License](https://img.shields.io/badge/-Apache%202.0-blue.svg)](https://opensource.org/s/Apache-2.0)
```

Comment:
The link is broken, should be https://opensource.org/licenses/Apache-2.0 or http://www.apache.org/licenses/LICENSE-2.0